### PR TITLE
fix: set `dqpl-toast` to `display:flex`

### PR DESCRIPTION
**Pull request by jkodu**
Was fixing a css issue in cauldron react and then noticed this as well. So putting in a fix.

Closes Issue: https://github.com/dequelabs/cauldron-react/issues/217

### CODE REVIEW

File: lib/components/toasts/style.less [hunk 1]
```diff
@@ -1,6 +1,7 @@
 @import "../../variables.less";
 
 @{prefix}toast {
+  display: flex;
```
Comment (schne324) [inline lib/components/toasts/style.less]: @jkodu this is already happening [below via the `.display-flex()` mixin](https://github.com/dequelabs/pattern-library/pull/128/files#diff-764ba649d62426b192e33d3c9901c784L11)

Comment (schne324) [reply lib/components/toasts/style.less]: **NOTE**: you have to expand the visible lines in order to see it 😄

Comment (jkodu) [reply lib/components/toasts/style.less]: Good catch @schne324.

I debugged and found that the `display:flex` actually happens due to the cascading style coming from `.dqpl-fadein-setup`.

I realised the actual fix is to swap `.dqpl-fadein-setup` with `.dqpl-fade-in-flex`. 

That fix is not within `pattern-library` repository. So I will close this & look into putting a fix in `cauldron-react`.

image

**Comment by jkodu** [comment]
Closing PR as per comment here - https://github.com/dequelabs/pattern-library/pull/128#discussion_r334575907

File: lib/components/toasts/style.less (hunk 1 above)
Comment (schne324) [reply lib/components/toasts/style.less]: ahah! `dqpl-fade-setup` is the culprit - good find!

**Pull request closed without merging**